src/clients/sound.ts

```diff
@@ -143,15 +143,15 @@ const init = async () => {
       return !!matchedNFT;
     });
     filteredAPITracks.forEach(apiTrack => {
-      if (apiTrack.tracks.length > 1) {
+      if (apiTrack.track.length > 1) {
```

Comment:
@Geimaj did u confirm this is still an array right, even though the name is now singular?

Reply:
This is actually returning an object not an array now, so I added the necessary changes